### PR TITLE
Skip win sign when rc

### PR DESCRIPTION
**Pull request by nop33**
- https://github.com/alephium/alephium-frontend/issues/812

### CODE REVIEW

File: .github/workflows/release-desktop.yml [hunk 1]
```diff
@@ -56,6 +56,13 @@ jobs:
           name: notarization-error-log
           path: ${{ github.workspace }}/apps/desktop-wallet/notarization-error.log
 
+      - name: Skip signing for rc versions
+        if: contains(github.event.ref, '-rc.')
+        uses: jaywcjlove/github-action-package@v2.0.0
+        with:
+          path: apps/desktop-wallet/package.json
+          unset: build.win.sign
```
Comment (mvaivre) [inline .github/workflows/release-desktop.yml:64]: pretty cool

File: .github/workflows/release-desktop.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Release desktop wallet
 
 on:
   push:
-    tags: ['alephium-desktop-wallet@[0-9]+.[0-9]+.[0-9]+*']
+    tags: ['alephium-desktop-wallet@[0-9]+.[0-9]+.[0-9]+', 'alephium-desktop-wallet@[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+']
```
Comment (mvaivre) [inline .github/workflows/release-desktop.yml:5]: Nit: I guess both patterns could be merged to one by setting the last part as an optional group (to be tested):

`alephium-desktop-wallet@[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?`

**Review by mvaivre — APPROVED** [review]
Approving because I only have on nit suggestion! 👏

File: .github/workflows/release-desktop.yml (hunk 2 above)
Comment (nop33) [reply .github/workflows/release-desktop.yml]: pushed

**Pull request merged**